submissions/trostinetska/hooli-style-popup/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="author" content="Larysa Trostinetska">
+    <meta name="description" content="popup">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <ul class="crumbs">
+                <li class="crumb"><a href="#" tabindex="0">Mail</a></li>
```

Comment:
By default, all interactive elements have a focus. no need to add `tabindex` attribute unless you want to change the default order.

Reply:
Thanks a lot! It was done.